packages/camera/android/src/main/java/io/flutter/plugins/camera/CameraPlugin.java

```diff
@@ -202,6 +202,12 @@ public void onMethodCall(MethodCall call, final Result result) {
           camera.takePicture((String) call.argument("path"), result);
           break;
         }
+      case "prepareForVideoRecording":
+        {
+          // not yet implemented for Android
```

Comment:
Would this need to be implemented on Android? If it does, this would require a `// Todo: ...`If not, I would include that this method does nothing on Android in the documentation on the dart side.

Reply:
I haven't tried the camera on Android, but if it would only be required there if it suffers from the same freezing issue.